exercises/isogram/example.rb

```diff
@@ -0,0 +1,10 @@
+module BookKeeping
+  VERSION = 6
+end
+
+class Isogram
+  def self.is_isogram?(str)
+    downcased_str = str.downcase
```

Comment:
`example.rb` should be a working solution to the problem.
If this is a working solution there's something wrong with the tests.